webapp/components/Comment.vue

```diff
@@ -38,14 +38,14 @@
       </div>
       <div v-show="!openEditCommentMenu">
         <content-viewer
-          v-if="comment.content.length < 400"
+          v-if="$filters.removeHtml(comment.content).length < 180"
           :content="comment.content"
           class="padding-left"
         />
         <div v-else class="show-more-or-less-div">
           <content-viewer
             v-if="isCollapsed"
-            :content="$filters.truncate(comment.content, 400)"
+            :content="$filters.truncate(comment.content, 180)"
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-09-23T10:18:08Z" title="Monday, September 23rd 2019, 12:18:08 pm +02:00">Sep 23, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -38,14 +38,14 @@
       </div>
       <div v-show="!openEditCommentMenu">
         <content-viewer
-          v-if="comment.content.length < 400"
+          v-if="$filters.removeHtml(comment.content).length < 180"
           :content="comment.content"
           class="padding-left"
         />
         <div v-else class="show-more-or-less-div">
           <content-viewer
             v-if="isCollapsed"
-            :content="$filters.truncate(comment.content, 400)"
+            :content="$filters.truncate(comment.content, 180)"
```


Please refactor the 180 to be not such a magic number. You could put it into `constants/comments.js`.

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-09-23T10:33:49Z" title="Monday, September 23rd 2019, 12:33:49 pm +02:00">Sep 23, 2019</time>_

---


I agree with @roschaefer . 😍